### PR TITLE
Fixes corporate security uniforms not being selectable

### LINKED ISSUE #5174: 'Security uniform, corporate' cannot be selected in loadouts
When you try to click on it to add it, nothing happens.

### PR DESCRIPTION
**Pull request by LittleBigKid2000**
Fixes #5174

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Did you actually *test* this? "cause, I strongly doubt this will fix it.

File: code/modules/client/preference/loadout/loadout_uniform.dm [hunk 1]
```diff
@@ -107,7 +107,7 @@
 	allowed_roles = list("Head of Security", "Warden", "Detective", "Security Officer", "Security Pod Pilot")
 
 datum/gear/uniform/secorporate
```
Comment (Fox-McCloud) [inline code/modules/client/preference/loadout/loadout_uniform.dm]: and this is why.

**Comment by FlattestGuitar** [comment]
Classy, me.

**Comment by LittleBigKid2000** [comment]
Yes, Fox, I did test it. I could select it after the change.

File: code/modules/client/preference/loadout/loadout_uniform.dm [hunk 2]
```diff
@@ -106,8 +106,8 @@
 	path = /obj/item/clothing/under/rank/security/formal
 	allowed_roles = list("Head of Security", "Warden", "Detective", "Security Officer", "Security Pod Pilot")
 
-datum/gear/uniform/secorporate
-	display_name = "security uniform, corporate "
+/datum/gear/uniform/secorporate
```
Comment (Fox-McCloud) [inline code/modules/client/preference/loadout/loadout_uniform.dm]: this still isn't right.

` /datum/gear/uniform/sec/secorporate`

**Comment by LittleBigKid2000** [comment]
It's @FlattestGuitar's fault, not mine

**Comment by FlattestGuitar** [comment]
I don't know of how exactly I goofed these up so hard. Blame lack of sleep and 800 more lines to type out.

**Pull request merged**